mkosi/__init__.py

```diff
@@ -4100,8 +4098,12 @@ def run_verb(args: Args, images: Sequence[Config], *, resources: Path) -> None:
                         package_dir=Path(package_dir),
                     )
 
-            if args.auto_bump:
-                bump_image_version()
+        if args.auto_bump:
+            bump_image_version()
```

Comment:
Is this changing indentation here because it doesn't need the temporary directories? That makes sense, but is a bit unexpected in the commit.

Reply:
Yeah exactly that, I was too lazy to make it a separate commit.